content/en/docs/pipelines/tutorials/benchmark-examples.md

```diff
@@ -128,7 +128,7 @@ In a count plot, the x-axis represents the possible run status: success or fail;
 and the y-axis shows how many runs fall into certain status respectively.
 
 In a distribution plot, both histogram plot and rug plot are shown. In addition,
-it is also possible to show KDE (Kernel Density Estimate) plot. If a KDE plot is
+it is also possible to show a KDE (Kernel Density Estimate) plot. If the KDE plot is
 desirable, use 'kde=True' in the distplot() method.
```

Comment:
Can you check the use of "code font" - singe backticks (vs quotation marks) for parameters and functions/methods, etc?  (Reference - [Google docs style guide: Code in text](https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font)).

Line 131:

```suggestion
desirable, use `kde=True` in the `distplot()` method.
```

Reply:
- Line 50:

```diff
- CREATE, GET, DELETE, and LIST methods of the pipeline, pipeline version
+ `CREATE`, `GET`, `DELETE`, and `LIST` methods of the pipeline, pipeline version
```

- Line 76:

```diff
-   1. Records the latency of CREATE, GET, DELETE.
+   1. Records the latency of `CREATE`, `GET`, `DELETE`.
```

- Line 80:

```diff
1. In the benchmark script, enter the correct values for host,
-   pipeline_file_url, num_runs, run_status_polling_interval_sec in the
+   `pipeline_file_url`, `num_runs`, `run_status_polling_interval_sec` in the
   benchmark script.
```

- Lines 92, 94:

```diff
-       **NOTE**: Do not use the value 'https://storage.cloud.google.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml'
+      **NOTE**: Do not use the value `https://storage.cloud.google.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml`
     when running your benchmarks. Addresses that start with
-      'storage.cloud.google.com' incur a redirect that doesn't work well with
+      `storage.cloud.google.com` incur a redirect that doesn't work well with
        ...
```

- Line 114:

```diff
- Each node pool has three nodes of machine type 'n1-standard-8'.
+ Each node pool has three nodes of machine type `n1-standard-8`.
```